pkgs/development/python-modules/pyopenssl/default.nix

```
@@ -39,6 +46,8 @@ buildPythonPackage rec {
     sphinx-rtd-theme
   ];
 
+  propagatedBuildInputs = [ typing-extensions ];
```

```suggestion
  dependencies = [ typing-extensions ];
```
and make this conditional to `pythonOlder "3.13"`